speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/exitmenu.cpp

```diff
@@ -20,18 +20,19 @@
 #include <tgfclient.h>
 
 #include "legacymenu.h"
+#include "ConfigEnums.h"
 
 #include "exitmenu.h"
+#include "SaveMenu.h"
 #include "mainmenu.h"
 
-
 static void *MenuHandle = NULL;
 
-static void 
-onAcceptExit(void * /* dummy */)
+static void
+OnAcceptExit(void * /* dummy */)
```

Comment:
Don't change speed dreams methods without any reason

Reply:
not even when we change it to work they way we want to use it?

Reply:
nvm, thought I had edited this function, I had not.